rustls/src/msgs/base.rs

```diff
@@ -118,6 +120,41 @@ impl PayloadU8 {
     }
 }
 
+// Format an iterator of u8 into a hex string
+pub(super) fn hex<'a>(
```

Comment:
Style nit: I'd prefer to have `hex()` somewhere at the bottom (utility function, fairly simple, unlikely to change often) and the `Payload*` impls near each type's definition (below other impls for the same type).

Reply:
👍🏻 Done!